deployments/gpu-operator/templates/clusterrole.yaml

```diff
@@ -147,10 +147,8 @@ rules:
   - list
   - watch
 - apiGroups:
-  - ""
   - coordination.k8s.io
   resources:
-  - configmaps
```

Comment:
why was this removed? did we add this RBAC somewhere else?

Reply:
Yeah, this is a duplicate. The configMaps permission already exists in the manifest file